subsys/fs/littlefs_fs.c

```diff
@@ -860,6 +897,42 @@ static int littlefs_mount(struct fs_mount_t *mountp)
 	return ret;
 }
 
+#if defined(CONFIG_FILE_SYSTEM_MKFS)
+
+static int littlefs_mkfs(struct fs_mount_t *mountp)
+{
+	int ret = 0;
+	struct fs_littlefs *fs = mountp->fs_data;
+
+	/* Create and take mutex. */
+	k_mutex_init(&fs->mutex);
+	fs_lock(fs);
```

Comment:
What will happen here when LittleFS at given mount point is already mounted, which means that it has already been initialized?

Reply:
Ah sorry, this is covered by `validate_mp`